ff-parser/src/main/java/org/uniprot/core/flatfile/parser/impl/cc/CcLineModelListener.java

```diff
@@ -601,9 +614,13 @@ public void exitCc_subcellular_location(CcLineParser.Cc_subcellular_locationCont
                             ccSubcellularLocationValueContext =
                                     locEvi.cc_subcellular_location_value();
 
-                    locationObject.orientation = new CcLineObject.LocationValue();
-                    locationObject.orientation.value =
-                            ccSubcellularLocationValueContext.cc_subcellular_words().getText();
+                    locationObject.setOrientation(new LocationValue());
```

Comment:
I think the code would be easier to read if you create the LocationValue, set the orientation value and after that you set it to the locationObject. What do you think?